src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -210,7 +211,7 @@ private int getInventorySlotContainItem(final Item itemIn)
      */
     public boolean addItemStackToInventory(@Nullable final ItemStack itemStackIn)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)